packages/astro/test/astro-dynamic.test.js

```diff
@@ -32,24 +32,26 @@ describe('Dynamic components', () => {
     expect(js).to.include(`value:"(max-width: 600px)"`);
   });
 
-  it.skip('Loads pages using client:only hydrator', async () => {
+  it('Loads pages using client:only hydrator', async () => {
     const html = await fixture.readFile('/client-only/index.html');
     const $ = cheerio.load(html);
 
     // test 1: <astro-root> is empty
     expect($('<astro-root>').html()).to.equal('');
+    const script = $('script').text();
+    console.log(script);
 
     // Grab the svelte import
-    const exp = /import\("(.+?)"\)/g;
-    let match, svelteRenderer;
-    while ((match = exp.exec(result.contents))) {
-      if (match[1].includes('renderers/renderer-svelte/client.js')) {
-        svelteRenderer = match[1];
-      }
-    }
+    // const exp = /import\("(.+?)"\)/g;
```

Comment:
Some other tests had a note about this being hard to locate with the move to Vite, along with a TODO. Maybe add one here?